### ISSUE TITLE
Chart version 1.6.0 has disappeared from repository

### ISSUE THREAD

**Issue by dalbani** [zalando/postgres-operator#1374]
It looks like the chart version 1.6.0 has disappeared from the Helm chart repository.
See <https://opensource.zalando.com/postgres-operator/charts/postgres-operator/index.yaml>.
I suppose it's related to the recent release of version 1.6.1.
Files related to chart 1.6.0 have been *replaced* by the ones for 1.6.1 in commit <https://github.com/zalando/postgres-operator/commit/3962e71ddd897e9aaa28c9b00e5e17e0d8602ead#diff-4093e6f34f5cb73eb381d5479941955afe97d343029d89aa9fa69ca20838b0a8>.
In the meantime we have switched to a URL like `https://raw.githubusercontent.com/zalando/postgres-operator/v1.6.0/charts/postgres-operator`.
Could you restore the 1.6.0 chart so we can use a single URL containing all versions?
Thanks!

**Comment by FxKu** [comment]
Thanks for raising this @dalbani . First I thought it's fine if we just get rid of it, because of the bugs. But of course some of you depend on it. So we put back in place incl. bugfixes.

**Comment by dalbani** [comment]
@FxKu  
Thanks for the quick fix.
Not a showstopper but I noticed that the `.tgz` for 1.6.0 contains itself:
![image](https://user-images.githubusercontent.com/1470270/108498958-e4e42e00-72ad-11eb-9321-13681182790f.png)

**Comment by dalbani** [comment]
@FxKu  
Oh, and what are those fixes by the way?

**Comment by FxKu** [comment]
ah damn, one `helm package` command to much. See #1351. Just typos and missing options.

**Comment by FxKu** [comment]
package is now updated and should not contain the extra archive

**Comment by reinvantveer** [comment]
Thanks!

**Comment by GMartinez-Sisti** [comment]
Hi! 

Sorry to comment on a closed issue but I noticed that the current 1.6.0.gz is not the same as the previous one, also includes posgres-operator-ui, and now we have changes that were not supposed to happen.

I would like to recomend that released versions should not be changed, as that might break deployments where folks are using IAC or GitOps and constantly ensuring that the config matches the desired ones.

These are the changes (according to our configs):

```yaml
diff --git a/source/kubernetes/output/aws-development.yaml b/source/kubernetes/output/aws-development.yaml
index 9e48f35..6971e46 100644
--- a/source/kubernetes/output/aws-development.yaml
+++ b/source/kubernetes/output/aws-development.yaml
@@ -97406,7 +97406,7 @@ spec:
       serviceAccountName: postgres-operator-ui
       containers:
         - name: "service"
-          image: "registry.opensource.zalan.do/acid/postgres-operator-ui:v1.6.0"
+          image: "registry.opensource.zalan.do/acid/postgres-operator-ui:v1.6.1"
           imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 8081
@@ -97559,15 +97559,6 @@ rules:
   - services
   verbs:
   - create
-# to run privileged pods
-- apiGroups:
-  - extensions
-  resources:
-  - podsecuritypolicies
-  resourceNames:
-  - privileged
-  verbs:
-  - use
 ---
 # Source: postgres-operator/templates/clusterrole.yaml
 apiVersion: rbac.authorization.k8s.io/v1
@@ -97772,15 +97763,6 @@ rules:
   verbs:
   - get
   - create
-# to grant privilege to run privileged pods
-- apiGroups:
-  - extensions
-  resources:
-  - podsecuritypolicies
-  resourceNames:
-  - privileged
-  verbs:
-  - use
 ---
 # Source: postgres-operator/templates/clusterrolebinding.yaml
 apiVersion: rbac.authorization.k8s.io/v1
@@ -97813,7 +97795,7 @@ metadata:
     app.kubernetes.io/instance: postgres-operator
 data:
   pod_service_account_name: postgres-pod
-  docker_image: registry.opensource.zalan.do/acid/spilo-13:2.0-p2
+  docker_image: registry.opensource.zalan.do/acid/spilo-13:2.0-p4
   enable_crd_validation: "true"
   enable_lazy_spilo_upgrade: "false"
   enable_pgversion_env_var: "true"
@@ -97859,7 +97841,6 @@ data:
   aws_region: eu-west-1
   enable_ebs_gp3_migration: "false"
   logical_backup_docker_image: registry.opensource.zalan.do/acid/logical-backup:v1.6.0
-  logical_backup_job_prefix: logical-backup-
   logical_backup_provider: s3
   logical_backup_s3_access_key_id: ""
   logical_backup_s3_bucket: my-bucket-url
@@ -97879,7 +97860,7 @@ data:
   connection_pooler_default_cpu_request: 500m
   connection_pooler_default_memory_limit: 100Mi
   connection_pooler_default_memory_request: 100Mi
-  connection_pooler_image: registry.opensource.zalan.do/acid/pgbouncer:master-9
+  connection_pooler_image: registry.opensource.zalan.do/acid/pgbouncer:master-14
   connection_pooler_max_db_connections: "60"
   connection_pooler_mode: transaction
   connection_pooler_number_of_instances: "2"
@@ -97905,7 +97886,7 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: af184716bf1d0fa076e51d70ed4545e00010e6f0697278bbcaa545b5c5751970
+        checksum/config: 194345c1cb4c32158985b683368821ba41fe03b7d369dccdf2886867ff5b23d9
       labels:
         app.kubernetes.io/name: postgres-operator
         app.kubernetes.io/instance: postgres-operator
@@ -97927,6 +97908,11 @@ spec:
           requests:
             cpu: 100m
             memory: 250Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          runAsUser: 1000
       affinity:
         {}
       nodeSelector:
```

Keep up the good work!
